### DISCUSSION TITLE
How to use ImageMagick for batch conversion *.jpg *.pdf

### DISCUSSION THREAD

**Discussion by tuxedotje** [ImageMagick/ImageMagick#5230] [Help] [answered]
Have a folder of .jpg files. Tried to <convert *.jpg *.pdf> . ImageMagick is installed. But terminal  closes/crashes after some minutes. 
[convert  filename.jpg to filename.pdf] (One file. not batch) does go well.

**Comment by fmw42** [comment, marked as answer]
You cannot have wild cards for both input and output.  You should use mogrify to convert a folder of images. See https://imagemagick.org/Usage/basics/#mogrify

**Reply by tuxedotje** [reply]
Thanks for this clear answer.
